sourcegraph/sourcegraph.proto

```diff
@@ -1835,6 +1845,14 @@ service Auth {
 			get: "/auth/identify"
 		};
 	};
+
+	// GetPermissions returns the currently authenticated user's
+	// authorization levels on the client.
+	rpc GetPermissions(pbtypes.Void) returns (UserPermissions) {
+		option (google.api.http) = {
+			get: "/auth/verify"
```

Comment:
Is "/auth/verify" intended or a copy-paste error? It doesn't seem to match the method namely, the way all others do.